test/e2e/flowaggregator_test.go

```diff
@@ -22,94 +22,85 @@ import (
-    sourcePodNamespace: antrea-test
-    sourceNodeName: k8s-node-master
-    destinationPodName: perftest-b
-    destinationPodNamespace: antrea-test
-    destinationNodeName: k8s-node-master
-    destinationClusterIPv4: 10.103.234.179
-    destinationServicePortName: antrea-test/perftest-b:
-    ingressNetworkPolicyName: test-networkpolicy-ingress
-    ingressNetworkPolicyNamespace: antrea-test
-    egressNetworkPolicyName: test-networkpolicy-egress
-    egressNetworkPolicyNamespace: antrea-test
+	DATA RECORD-0:
+	flowStartSeconds: 1608254105
+	flowEndSeconds: 1608254112
+	sourceTransportPort: 40518
+	destinationTransportPort: 5201
+	protocolIdentifier: 6
+	packetTotalCount: 752801
+	octetTotalCount: 32611437907
+	packetDeltaCount: 546687
+	octetDeltaCount: 23669907884
+	sourceIPv4Address: 10.10.0.26
+	destinationIPv4Address: 10.10.0.27
+	reversePacketTotalCount: 551996
+	reverseOctetTotalCount: 28705000
+	reversePacketDeltaCount: 399085
+	reverseOctetDeltaCount: 20753584
+	sourcePodName: perftest-a
+	sourcePodNamespace: antrea-test
+	sourceNodeName: k8s-node-master
+	destinationPodName: perftest-b
+	destinationPodNamespace: antrea-test
+	destinationNodeName: k8s-node-master
+	destinationServicePort: 5201
+	destinationServicePortName:
+	ingressNetworkPolicyName: test-flow-aggregator-networkpolicy-ingress
+	ingressNetworkPolicyNamespace: antrea-test
+	egressNetworkPolicyName: test-flow-aggregator-networkpolicy-egress
+	egressNetworkPolicyNamespace: antrea-test
+	destinationClusterIPv4: 0.0.0.0
+	originalExporterIPv4Address: 10.10.0.1
+	originalObservationDomainId: 2134708971
 
 Intra-Node: Flow record information is complete for source and destination e.g. sourcePodName, destinationPodName
 Inter-Node: Flow record from destination Node is ignored, so only flow record from the source Node has its K8s info e.g., sourcePodName, sourcePodNamespace, sourceNodeName etc.
 AntreaProxy enabled (Intra-Node): Flow record information is complete for source and destination along with K8s service info such as destinationClusterIP, destinationServicePort, destinationServicePortName etc.
 AntreaProxy enabled (Inter-Node): Flow record from destination Node is ignored, so only flow record from the source Node has its K8s info like in Inter-Node case along with K8s Service info such as destinationClusterIP, destinationServicePort, destinationServicePortName etc.
 */
 
-func TestFlowExporter(t *testing.T) {
-	// Should I add skipBenchmark as this runs iperf?
+const (
+	ingressNetworkPolicyName   = "test-flow-aggregator-networkpolicy-ingress"
+	egressNetworkPolicyName    = "test-flow-aggregator-networkpolicy-egress"
+	collectorCheckDelay        = 1 * time.Second
+	expectedNumTemplateRecords = 1
+	// Single iperf resulting in two connections with separate ports. Suspecting second flow to be control flow to exchange
+	// stats info. As 5s is export interval and iperf traffic runs for 10s, we expect about 4 records exporting to the flow aggregator.
```

Comment:
That's correct, iperf uses a "control" connection

suugestion: Single iperf run results in two connections with separate ports (control connection and actual data connection).

Reply:
Thanks, addressed.